src/System.Private.CoreLib/src/System/Runtime/Serialization/FormatterServices.cs

```diff
@@ -2,28 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-/*============================================================
-**
-**
-**
-** Purpose: Provides some static methods to aid with the implementation
-**          of a Formatter for Serialization.
-**
-**
-============================================================*/
-
-using System;
-using System.Reflection;
-using System.Collections;
-using System.Collections.Generic;
-using System.Security;
 using System.Runtime.CompilerServices;
-using System.Runtime.Versioning;
-using System.Threading;
-using System.IO;
-using System.Text;
-using System.Globalization;
-using System.Diagnostics;
 
 namespace System.Runtime.Serialization
 {
```

Comment:
There is nothing keeping the `FormatterServices` alive, so it will get trimmed in the shipping bits.

Either delete this file and see whether anybody will notice; or keep it explicitly pinned in https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/ILLinkTrim.xml . I would be ok with deleting it.

Reply:
File removed :-)